boxes_post_processing.py

```diff
@@ -0,0 +1,249 @@
+import sys
+import os 
+import pandas as pd
+import time
+import numpy as np
+from gz.math7 import Quaterniond
+import matplotlib.pyplot as plt
+import csv
+import time
+
+
+DIRECTORY_NAMES = ["BENCHMARK_boxes_dt"]
+
+class PostProcessing:
+       
+       def __init__(self, test_name: str):
+           self.m = 10
+           self.g = 9.8
+           self.box_x = 0.1
+           self.box_y = 0.4
+           self.box_z = 0.9
+           Ixx = self.m/12.0 * (self.box_y**2 + self.box_z**2)
+           Iyy = self.m/12.0 * (self.box_z**2 + self.box_x**2)
+           Izz = self.m/12.0 * (self.box_x**2 + self.box_y**2)
+           self.I = np.diag([Ixx, Iyy, Izz])
+           self.sim_duration = 10
+           
+           timestr = time.strftime("%Y%m%d-%H%M%S")
+           metrics_filename = test_name + timestr + ".csv"
+           metrics_path = os.path.join("~", "benchmark", "test_results", metrics_filename)
```

Comment:
this line fails for me because I don't have `benchmark` directly in my `$HOME` folder. do you want to use the same folder logic as [mcap_to_csv.py](https://github.com/scpeters/benchmark/blob/master/tools/mcap_to_csv.py#L4-L36)?

Reply:
yeah, I have few changes to push which should resolve the issues.